### PR TITLE
chore: ui fixes for adding pop-up + copy changes for BIP

### PR DESCRIPTION
**Pull request by PikkaPikkachu**
## Context
This PR makes some copy fixes and adds pop-up on the download csv button, so that users are aware that they should download the csv after some time, so that recipients column is visible. 

We also make styling fixes to the feedback modal as recommended by Feli.

## Approach

1. Made the relevant UI/ copy changes.
2. Styling changes for feedback modal

## Before & After Screenshots
  
https://github.com/opengovsg/letters/assets/25703976/2d49dd85-2a12-42e3-8021-e231fe17247a

## Feedback modal before vs after 
Screenshot 2023-08-10 at 12 50 19 PM

Screenshot 2023-08-10 at 12 49 11 PM

### CODE REVIEW

File: frontend/src/features/dashboard/components/IssuedBatchesTable.tsx [hunk 1]
```diff
@@ -123,7 +123,38 @@ export const IssuedBatchesTable = (): JSX.Element => {
...
+                          width="250px"
+                        >
+                          Recommended: Please download csv a few minutes after
+                          letters are issued.
+                        </PopoverBody>
+                      </Stack>
+                    </PopoverContent>
+                  </Popover>
+                </Box>
+              </HStack>{' '}
```
Comment (gweiying) [inline frontend/src/features/dashboard/components/IssuedBatchesTable.tsx:156]: non-blocking nit, this whitespace might be leftover from linting?

File: frontend/src/features/dashboard/components/IssuedBatchesTable.tsx [hunk 2]
```diff
@@ -123,7 +123,38 @@ export const IssuedBatchesTable = (): JSX.Element => {
...
+                    <PopoverContent bg="base.content.strong" width="250px">
+                      <Stack whiteSpace="normal">
+                        <PopoverBody
+                          textAlign={'center'}
+                          textStyle={'body-2'}
+                          textColor={'white'}
+                          textOverflow={'wrap'}
+                          width="250px"
+                        >
+                          Recommended: Please download csv a few minutes after
```
Comment (gweiying) [inline frontend/src/features/dashboard/components/IssuedBatchesTable.tsx:149]: nit, should we capitalize CSV here to be standardized with all the rest of the references to CSV files in other parts of the frontend?

File: frontend/src/features/issue/components/BulkIssueSettingsCard.tsx [hunk 3]
```diff
@@ -89,17 +89,17 @@ export const BulkIssueSettingsCard = ({
...
               <Radio value={CitizenNotificationMethod.SMS} size={'sm'}>
                 <Text textStyle="subhead-2" textColor={'grey.800'}>
-                  Notify via LetterSG SMS
+                  LetterSG SMS
                 </Text>
               </Radio>
               <Radio value={CitizenNotificationMethod.EMAIL} size={'sm'}>
                 <HStack>
-                  <Text textStyle="subhead-2">Notify via LetterSG email</Text>
+                  <Text textStyle="subhead-2"> LetterSG email</Text>
```
Comment (gweiying) [inline frontend/src/features/issue/components/BulkIssueSettingsCard.tsx:97]: Nit, some whitespace infront of ` LetterSG`

**Review by gweiying — APPROVED** [review]
tiny nits, but overall looks good! just curious, there're no actual toasts added right?

**Comment by PikkaPikkachu** [comment]
> tiny nits, but overall looks good! just curious, there're no actual toasts added right?

Ooops. Yes it's pop-up not a toast :P

**Pull request merged**